### ISSUE TITLE
[EXE] Horses_vs_Humans Classification

### ISSUE THREAD

**Issue by Overlrd** [gimseng/99-ML-Learning-Projects#197]
#### Learning Goals

- Learn Convolutional Neural Networks in  Tensorflow
- Learn Multi-dimentional Image Classification
- Learn Image Augmentation 

### Exercise Statement

Write a Convolutional Neural Network for binary classification on the horse_vs_human dataset

### Prerequisites 
- Computer vision fundamentals including simple neural networks

### Data source/summary:
Horses or Humans is a dataset of 300×300 images, created by Laurence Moroney, that is licensed CC-By-2.0 for anybody to use in learning or testing computer vision algorithms.

The set contains 500 rendered images of various species of horse in various poses in various locations. It also contains 527 rendered images of humans in various poses and locations. Emphasis has been taken to ensure the diversity of humans, and to that end, there are both men and women as well as Asian, Black, South Asian, and Caucasians present in the training set. The validation set adds 6 different figures of different gender, race, and poses to ensure breadth of data.

For more info http://www.laurencemoroney.com/horses-or-humans-dataset/

### (Optional) Suggest/Propose Solutions
I have the solution using Tensorflow, will be happy to create pull request to include the exercise statement/solution]

### (Optional) Further Links/Credits to Relevant Resources:
This exercise and solution's proposal came from a lab session from [The Tensorflow Developer Professional Certificate Course On Coursera](https://www.coursera.org/professional-certificates/tensorflow-in-practice)

**Comment by DhruvSanghvi2002** [comment]
I would love to contribute over the project, pls assign me this issue
